<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RamificacionYAcotamiento</class>
 <widget class="QMainWindow" name="RamificacionYAcotamiento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>CMU Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ramificación Y Acotamiento</string>
  </property>
  <property name="locale">
   <locale language="Spanish" country="Mexico"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="tipo_entero">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Entero</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="tipo_mixto">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mixto</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="tipo_binario">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Binario</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de Planteamiento</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Número de variables (Máx 10.)</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Número de Restricciones</string>
    </property>
   </widget>
   <widget class="QSlider" name="numero_variables_slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="numero_restricciones_slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="valor_slider_variables">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="valor_slider_restricciones">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Variables</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="variables_tabla">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>1261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <row/>
    <column/>
   </widget>
   <widget class="QTableWidget" name="restricciones_tabla">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>1261</width>
      <height>321</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <row/>
    <column/>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Restricciones</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ttulo">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ramificación y Acotamiento</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="resolver_boton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>640</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolver</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="nuevo_planteamiento_action"/>
   </widget>
   <widget class="QMenu" name="menuMostrar_Ejemplo">
    <property name="title">
     <string>Mostrar Ejemplo</string>
    </property>
    <addaction name="ejemplo_action"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuMostrar_Ejemplo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="nuevo_planteamiento_action">
   <property name="text">
    <string>Nuevo Planteamiento</string>
   </property>
  </action>
  <action name="ejemplo_action">
   <property name="text">
    <string>Ejemplo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
